docs/development/versioning.md

```diff
@@ -92,3 +94,17 @@ As breaking releases happen about once every six months, this allows six to twel
 **In some cases, we may decide to adjust the deprecation period.**
 If retaining the deprecated functionality blocks other improvements to Polars, we may shorten the deprecation period to a single breaking release. This will be mentioned in the warning message.
 If the deprecation affects many users, we may extend the deprecation period.
+
+## Release frequency
+
+Polars does not have a set release schedule.
+We issue a new release whenever we feel like we have something new to offer that's valuable to our users.
+In practice, a new minor version is released about once every one or two weeks.
+
+### Breaking releases
+
+Over time, issues pop up that require a breaking change to address.
+When enough issues have accumulated, we issue a breaking release.
+
+So far, breaking releases have happened about once every three to six months.
+From this point on, we expect new major versions to be released about once every six months.
```

Comment:
Can we also add a sort of reassuring phrase that we expect the severity of the changes to reduce and be more on the periphery.

Reply:
I added in:

> The rate and severity of breaking changes will continue to diminish as Polars grows more solid.